group.py

```diff
@@ -2,4 +2,20 @@
 
 # Your code to go here...
 
-my_group =
+my_group = Jiayi21-JiefuMei-myx2021-thandi1908-SolemnShark871
+
+class Person:
+    def _init_(self, name, age, occupation ):
+        self.name = name
+        self.age = age
+        self.occupation = occupation
+        self.relationships = {}  #define relationship
+
+    def add_relationships(self, name, relationships):
+        
+
+
+Jill_relations = {Zalika: ""friend", John: "partner"}
```

Comment:
I think there's an unclosed quote here
```suggestion
Jill_relations = {Zalika: "friend", John: "partner"}
```